lib/kubeclient/common.rb

```diff
@@ -426,8 +422,8 @@ def load_entities
       @entities = {}
       fetch_entities['resources'].each do |resource|
         next if resource['name'].include?('/')
-        resource['kind'] = Kubeclient::Common::MissingKindCompatibility
-                           .resource_kind(resource['name']) if resource['kind'].nil?
+        resource['kind'] ||=
```

Comment:
This will also use the falback mapping if it's `false`.  Should never be a boolean, either a string or absent.  And if it could be `false`, using mapping is better.  => OK